experimental/generation/runbot/Startup.cs

```diff
@@ -45,12 +47,22 @@ public void ConfigureServices(IServiceCollection services)
             services.AddSingleton<AuthenticationConfiguration>();
 
             // register components.
-            ComponentRegistration.Add(new DialogsComponentRegistration());
-            ComponentRegistration.Add(new DeclarativeComponentRegistration());
-            ComponentRegistration.Add(new AdaptiveComponentRegistration());
-            ComponentRegistration.Add(new LanguageGenerationComponentRegistration());
-            ComponentRegistration.Add(new QnAMakerComponentRegistration());
-            ComponentRegistration.Add(new LuisComponentRegistration());
+            ComponentRegistration.Add(new DeclarativeComponentRegistrationBridge<DialogsBotComponent>());
+            ComponentRegistration.Add(new DeclarativeComponentRegistrationBridge<DeclarativeBotComponent>());
+            ComponentRegistration.Add(new DeclarativeComponentRegistrationBridge<AdaptiveBotComponent>());
+            ComponentRegistration.Add(new DeclarativeComponentRegistrationBridge<LanguageGenerationBotComponent>());
+            ComponentRegistration.Add(new DeclarativeComponentRegistrationBridge<QnAMakerBotComponent>());
+            ComponentRegistration.Add(new DeclarativeComponentRegistrationBridge<LuisBotComponent>());
+            ComponentRegistration.Add(new DeclarativeComponentRegistrationBridge<DynamicListBotComponent>());
```

Comment:
This works fine, but you only really needed to do this for your dynamic list bot component.